css/admin.upgrade.css

```
@@ -84,19 +92,23 @@
 	text-decoration: none;
 	transition-duration: .2sec;
 }
+
 .genesis-custom-blocks-pro .btn:hover {
```

```suggestion
.genesis-custom-blocks-pro .container .btn:hover {
```

This is just for consistency.

---

Sounds good